reference/catalog/products_catalog.v3.yml

```diff
@@ -7534,6 +7548,12 @@ components:
             type: number
           maximum: 1000
           minimum: 0
+        channels:
+          type: array
+          description: | 
+            An array of channel IDs to which this product is assigned.
```

Comment:
```suggestion
            An array of channel IDs that have the product assigned.
```

Reply:
@bc-andreadao @kristinapototska suggested this definition so I went with it. I can see if she is okay with the change.